jans-core/document-store/src/main/java/io/jans/service/document/store/service/OxDocument.java

```diff
@@ -0,0 +1,150 @@
+package io.jans.service.document.store.service;
+
+import java.io.Serializable;
+import java.util.Date;
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+
+import io.jans.orm.annotation.AttributeName;
+import io.jans.orm.annotation.DataEntry;
+import io.jans.orm.annotation.ObjectClass;
+import io.jans.orm.model.base.Entry;
+
+/**
+ * oxDocument
+ * 
+ * @author Shekhar L. Date: 01.10.2022
+ */
+
+
+@DataEntry(sortBy = { "displayName" })
+@ObjectClass(value = "oxDocument")
+@JsonInclude(Include.NON_NULL)
+public class OxDocument extends Entry implements Serializable {
+
+	private static final long serialVersionUID = -2812480357430436503L;
+
+	private transient boolean selected;
+
+	@AttributeName(ignoreDuringUpdate = true)
+	private String inum;
+
+
+	@AttributeName
+	private String displayName;
+
+	@AttributeName
+	private String description;
+
+	@AttributeName
+	private String document;
+	
+	@AttributeName
+	private Date creationDate;
+	
+	@AttributeName
+	private List<String> oxModuleProperty;
+
+	@AttributeName
+	private String oxLevel;
```

Comment:
This attribute is not present in jans. As well as other attributes refixed here as `ox*`. It must be changed. Please check existing schema in `jans_schema.json`. Lets re-use existing attributes.